include/dts-functions.sh

```diff
@@ -747,62 +747,34 @@ check_se_creds() {
   fi
 }
 
+semver() {
+  # return 1 if $1 < $2, 0 if $1 = $2 or -1 if $1 > $2
+
+  local valid1=
+  local valid2=
+  valid1="$(python3 -c "import semver; print(semver.Version.is_valid('$1'))")"
```

Comment:
It can also be used as a CLI tool, maybe that's more suitable here?  `python -m semver compare/check`

Reply:
Didn't know it was possible. Changed in cacba6ee9673989b3df7106cd3614261ab6f16a3
And I put everything in one function.